src/js/client/apply.js

```diff
@@ -102,7 +103,7 @@ function processForm($form, createForm) {
       const field = form.fields[fieldName];
 
       $row.removeClass('error');
-      $row.find('.error_msg').remove();
```

Comment:
I think this breaks the error handling on the individual application form.

Reply:
I've adopted a new strategy which is to replace the _inner_ HTML of a preexisting element rather than the whole element—this allows error elements to be different for normal forms and table forms